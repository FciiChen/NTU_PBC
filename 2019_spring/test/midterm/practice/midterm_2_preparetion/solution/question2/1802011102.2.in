co9I, uxRpF9foyY,gzeqkk2,jzRone
pSFH, wWHL,4pBU,Lo6uHHmg
hXtOYzYkq5, YlLzUCdWg3,N21K,kB2dCAX
OcUWcaym, RkcgIni5hS,Lc4r,MmgSj0dDub
bSGG5IkFTF, VLt6hz,S2iIWaFL,IYVGxr
R1Fl, LYoRvV,Z8Ymlyq,sNHka
BREAK
pSFH ,  CvGgRG
hXtOYzYkq5 ,  PHdhX
OcUWcaym ,  Xe2SZ
co9I ,  mZ6djPUhi
BREAK
