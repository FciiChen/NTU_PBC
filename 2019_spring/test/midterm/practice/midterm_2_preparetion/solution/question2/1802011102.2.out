co9I       uxRpF9foyY gzeqkk2    jzRone     mZ6djPUhi 
pSFH       wWHL       4pBU       Lo6uHHmg   CvGgRG    
hXtOYzYkq5 YlLzUCdWg3 N21K       kB2dCAX    PHdhX     
OcUWcaym   RkcgIni5hS Lc4r       MmgSj0dDub Xe2SZ     
bSGG5IkFTF VLt6hz     S2iIWaFL   IYVGxr     NULL      
R1Fl       LYoRvV     Z8Ymlyq    sNHka      NULL      
